doc/whats_new/v1.4.rst

```diff
@@ -230,6 +230,10 @@ Changelog
 - |Enhancement| :func:`sklearn.model_selection.train_test_split` now supports
   Array API compatible inputs. :pr:`26855` by `Tim Head`_.
 
+- |Fix| :class:`model_selection.ValidationCurveDisplay` now accepts passing a
+  list to the `param_range` parameter.
+  :pr:`27311` by :user:`Arturo Amor <ArturoAmorQ>`.
```

Comment:
I we plan to backport this to 1.3.1, shouldn't we directly put this changelog entry for 1.3.1 in `doc/whats_new/1.3.rst` instead?

Reply:
/cc @jeremiedbb

Reply:
indeed :)